test/numa/README.md

```diff
@@ -0,0 +1,129 @@
+# CRI Resource Manager - NUMA node tests
+
+## Prerequisites
+
+Install:
+- `docker`
+- `govm`
+- `jq`
+
+## Usage
+
+```
+[VAR=VALUE...] ./run.sh MODE
+```
+
+Get help on available `VAR=VALUE`'s with `./run.sh`.
+
+## Two modes: `test` and `play`
+
+`test` mode runs fast and, by default, cleans up everything after test
+run. The virtual machine with the contents will be lost. The output
+will include `Test verdict:`, and the exit status is zero if and only
+if the test passed.
+
+`play` mode runs slower and, by default, leaves the virtual machine
+running.
+
+Print help to see clean up, execution speed and other options for both
+modes.
+
+## Running from scratch and quick rerun in existing virtual machine
+
+The test will use `govm` virtual machine named in the `vm` environment
+variable. The default is `crirm-test-numa`. If a virtual machine with
+that name exists, the test will be run on it. Otherwise the test will
+create a virtual machine with that name from scratch. You can delete a
+virtual machine with `govm delete NAME`.
+
+If you want rerun the test many times, possibly with different test
+inputs or against different versions of `cri-resmgr`, either use the
+`play` mode or set `cleanup=0` in order to keep the virtual machine
+after each run. Then tests will run in the same single node cluster,
+and the test script will only delete running pods before launching new
+ones.
+
+## Testing locally built cri-resmgr and cri-resmgr from github
+
+If you make changes to `cri-resmgr` sources and rebuild it, you can
+force the test script to reinstall newly built `cri-resmgr` to
+existing virtual machine before rerunning the test:
+
+```
+cri-resource-manager$ make
+cri-resource-manager$ cd test/numa
+numa$ reinstall_cri_resmgr=1 speed=1000 ./run.sh play
+```
+
+You can also let the test script build `cri-resmgr` from the github
+master branch. This takes place inside the virtual machine, so your
+local git sources will not be affected:
+
+```
+numa$ reinstall_cri_resmgr=1 binsrc=github ./run.sh play
+```
+
+## Testing different Pods
+
+The test will instantiate Pods listed in the `pods` variable in
+JSON. You can print the default `pods` with `./run.sh help defaults`.
+
+The JSON list contains pod groups. Each group specifies template file,
+number of pods to be created with these specifications, and contents
+to the fields in template files (`CPU`, `CPULIM`, `MEM`, ... depends
+on the template).
+
+```
+{
+  "q": "guaranteed", // specifies the QoS class, or more precisely,
```

Comment:
nitpick: wouldn't `qos` or `qosclass` be more intuitive/easier to remember ? Or, since this specifies the pod template, wouldn't it be more intuitive to call this `template` or `podtemplate` ?